### DISCUSSION TITLE
Проект умер?

### DISCUSSION THREAD

**Discussion by ghost** [lirix360/ReadmangaGrabber#47] [Q&A] [answered]
Несколько месяцев не было обновлений + проект завис на v2 beta 7
Жалко потерять такой кладезь знаний, как актуальность кода этого проекта)

**Comment by lirix360** [comment, marked as answer]
Нет, не умер. С ридмангой все работает нормально. Мангалиб - кому как повезет.
Какие-то идеи есть, как будут время и настроение что-то будет сделано.
